Automattic/jetpack
# Protect: update new version of icon

Fixes #

#### Changes proposed in this Pull Request:

* Protect: update new version of icon

#### Jetpack product discussion


p1651503257875079-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use Storybook

JS Packages / Components / Product Icons | Plugins / Protect / Product Offer
-----|-----
<img width="1103" alt="Screen Shot 2022-05-02 at 4 13 01 PM" src="https://user-images.githubusercontent.com/77539/166310313-910ef7e2-c04c-450d-b9d7-9c659b71f44f.png"> | <img width="814" alt="image" src="https://user-images.githubusercontent.com/77539/166310435-48cdd2e5-b8fb-4ce2-95ae-f190a5699325.png">

* Check Protect plugin

Dashboard | Interstitial
------|-------
<img width="642" alt="Screen Shot 2022-05-02 at 4 16 02 PM" src="https://user-images.githubusercontent.com/77539/166310780-473929d8-33ad-42ab-a34f-82639cb2a945.png"> | <img width="599" alt="Screen Shot 2022-05-02 at 4 17 13 PM" src="https://user-images.githubusercontent.com/77539/166310925-a38dd9f0-4569-44c4-a501-f6dc63360681.png">

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202211721983844
